tests/cases/conformance/externalModules/umd9.ts

```diff
@@ -0,0 +1,14 @@
+// @module: commonjs

```

Comment:
I don't think this should be allowed with commonjs but only with umd?

Reply:
The output/resolution format of the program doesn't really influence this flag - *whether* you import something from a UMD module is irrelevant to *how* you might import it